scripts/assets/test_file_exemptions.textproto

```diff
@@ -646,6 +646,8 @@ exempted_file_path: "domain/src/main/java/org/oppia/android/domain/classify/rule
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/classify/rules/ratioinput/RatioInputModule.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/classify/rules/textinput/TextInputRuleModule.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgress.kt"
+exempted_file_path: "domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressListener.kt"
+exempted_file_path: "domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressModule.kt"
```

Comment:
This actually could be tested--I suggest adding a basic test suite to demonstrate that the binding is set up correctly & as expected by the module (we have a few other module tests in the codebase that you could look at for reference).

Reply:
Noted, thanks.